ogc/src/main/scala/geotrellis/server/ogc/wms/CapabilitiesView.scala

```diff
@@ -62,10 +63,38 @@ class CapabilitiesView(
         ) :: Nil
       )
 
+
+      val extendedCapabilities = {
```

Comment:
I believe I recall you saying somewhere offline that you were doing some research on the best way to represent extended capabilities in GT server. It looks like the end result is entirely static. Were there other solutions attempted? Could you talk a bit about the solutions you looked at for this and why you ultimately went with a static in code representation? (Mostly for our own future benefit when we inevitably revisit this later)

To be clear, since this feature is new to us, I think the solution here is fine for now -- I'm just trying to expose some of the context around the chosen implementation. 

We'll need to be vigilant about ensuring these definitions in WMS/WCS stay up to date as time goes on in case we add/remove additional capabilities. Perhaps a checklist item on the PR template would suffice as a reminder for now?

Reply:
It is static for now, because we need it only for two operations. 
I would say the only way to expose it dynamically is to add them into the application.conf file and to generate it from the application.conf description.

However, there is still an unsolved probelm (that I plan to work on today) - how to use them.

So this task basically covers XML semantics (since they are not described and can be arbitrary), implements static extra parameters support for two operations.

The next step would be to make use of them in GetCoverage and GetMap requests.
And only after that I think it makes sense to think a little about its generalization.

Reply:
Also - I am not sure that PR template makes a lot of sense. Do we need to create a template for each subproject than? or What is your suggestion? I don't expect these things to change a lot in the future :D

Reply:
Eh, lets wait to explore that (the PR suggestion) until the GetMap + equivalents are implemented. It'll be clearer then what would have to change if we were to add/remove support for one of these user-defined capabilities.